misc-scripts/xref_mapping/XrefParser/BaseParser.pm

```diff
@@ -1515,6 +1515,23 @@ sub get_meta_value {
 }
 
 
+######################################
+# Update info_type of an existing xref
+######################################
+sub _update_xref_info_type {
+  my ($self, $xref_id, $info_type, $dbi) = @_;
+
+  $dbi //= $self->dbi;
+
+  my $sth =  $dbi->prepare('UPDATE xref SET info_type=? where xref_id=?');
```

Comment:
I'd like for us to start caching these prepared statements in any oft-called utility functions within the lifetime of the parser.